geocodingsearchv7/client.go

```diff
@@ -24,6 +25,9 @@ type GeocodingService service
 // ReverseGeocodingService handles communication with reverse geocoding-related methods of the v7 HERE API.
 type ReverseGeocodingService service
 
+// BatchGeocoderService handles communication with batch geocoder-related methods of the v7 HERE API.
+type BatchGeocoderService service
```

Comment:
Since there is a GeocodingService and a ReverseGeocodingService, do you think there might be confusing for the consumer that there is no BatchReverseGeocodingService when there is a BatchGeocodingService?

Reply:
It's a lot easier to use it when there is only one service, i.e that BatchGeocodingService contains both Reverse and Forward Geocoding. It complicated the code that the one using the service has to write quite much which I would prefer not to do.

Reply:
Alright, good point!